frontend/src/App.js

```diff
@@ -16,6 +16,8 @@ function App() {
 
   const { data: currentUser } = useCurrentUser();
 
+  console.log(currentUser);
```

Comment:
Is this a necessary log? Also should revert changes to package-lock.json

Reply:
I need to remove the log, but I think the package-lock.json change is maybe automatic or something because I did not change that file.

Reply:
But i can remove it too